_assets/css/custom/_lists.scss

```diff
@@ -0,0 +1,3 @@
+.list-margin-unstyled {
+  list-style-type: none;
```

Comment:
Not sure if this accomplishes the same thing, but USWDS does provide an unstyled list class with `.usa-list.usa-list--unstyled`.

Reply:
Good catch @hursey013. I started by refactoring just about everything to unordered and ordered lists, but the `unstyled` uswds class also removes margins. I ended up just using margin classes anyway, so this custom rule can go away.